wordpress-mobile/WordPress-Android
# Add a fastlane action to create screenshot devices

**To Test:**
_BEWARE – THIS WILL OVERWRITE YOUR EXISTING SCREENSHOT GENERATION EMULATORS_

In the root of the project, run:

```
bundle install --path vendor/bundle
bundle exec fastlane android rebuild_screenshot_devices
```

There should be no side effects – the script is designed such that:

- Any non-screenshot emulators shouldn't be affected
- Any custom hardware profiles shouldn't be affected

The line count _looks_ high, but it's mostly the emulator and device profile definitions. For a bit more info on how this all works, check out [SCREENSHOT_DEVICE_SETUP.md](https://github.com/wordpress-mobile/WordPress-Android/blob/6d0a825622fda673e328a8a5893b272eb48b47c9/fastlane/SCREENSHOT_DEVICE_SETUP.md)
